frontend/app/views/notes/_template.html.erb

```diff
@@ -36,31 +36,12 @@
   </div>
 <% end %>
 
-
-
-<div id="template_note_summary"><!--
```

Comment:
We only want to define this template once -- it's "pathless" .. so I've moved it below into a conditional.